cypress.config.ts

```diff
@@ -0,0 +1,26 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
+  },
+  video: true,
+  viewportHeight: 1920,
+  viewportWidth: 1080,
+  screenshotOnRunFailure: true,
+  reporter: 'mochawesome',
+  reporterOptions: {
+    reportDir: 'raw_reports',
+    overwrite: false,
+    html: false,
+    json: true,
+  },
+  component: {
+    specPattern: 'src/**/*.spec.{js,ts,jsx,tsx}',
```

Comment:
Ensure that the `src` directory contains your component test specifications. This is important for Cypress to locate and execute the component tests.